openapi.yml

```diff
@@ -973,7 +973,9 @@ components:
             and table name, combined.
           type: string
         metadata:
-          $ref: "#/components/schemas/stream-level-metadata"
+          type: array
+          items:
+            $ref: "#/components/schemas/stream-level-metadata"
```

Comment:
note: i'm not sure why this is working because `metadata` should either be a single stream-level metadata (https://www.stitchdata.com/docs/developers/stitch-connect/api#stream--object) or an array of general metadata objects (https://www.stitchdata.com/docs/developers/stitch-connect/api#update-a-streams-metadata--arguments)

but defining that structure in the talos repo spec fails schema validation for the mock endpoints for a reason i cannot figure out